src/main/kotlin/message/ExceptionMessage.kt

```diff
@@ -0,0 +1,11 @@
+package message
+
+class ExceptionMessage {
+    companion object {
```

Comment:
```suggestion
object ExceptionMessage {
```
클래스 내부에 companion object만 활용할 것이라면, class 대신 object를 활용할 수 있어요 (이하 관련 내용 동일)

Reply:
동반 객체와 관련해 궁금한 것이 있습니다.
제 경우 이번 자동차 경주 미션 때와 마찬가지로 message를 갖는 상수들을 종류에 맞게 class로 묶어 하나의 class 에서 관리를 하는 방법으로 사용해 왔는데.
해당 message를 사용하는 클래스 내에서 동반 객체를 생성하여 상수를 사용하는 방법과 위 방법 중 어느 것이 관리, 코드 작성 하는데에 이점이 있을까요?? 리뷰어님과 실제 현업에서는 주로 어떤 방법으로 상수를 갖는 동반 객체를 관리하나요?

Reply:
대체로 클래스는 최소한 하나 이상을 캡슐화를 해야합니다.
그렇지 않으면, 클래스를 사용하는 것 보다는 object를 사용하는 것을 권장합니다.
특히 상수값같은 경우에는, 여러 클래스에서 접근하는 경우에는 별도의 object를 만들어서 관리하고 있으며, 클래스 내에서만 사용하는 상수값이라면 companion object에 관리하고 있습니다.